client/js/keybinds.js

```diff
@@ -58,6 +58,28 @@ Mousetrap.bind([
 	channels.eq(target).click();
 });
 
+Mousetrap.bind([
+	"alt+shift+up",
+	"alt+shift+down",
+], function(e, keys) {
+	const lobbies = sidebar.find(".lobby");
+	const index = lobbies.index(lobbies.filter(".active"));
+	const direction = keys.split("+").pop();
+	let target;
+
+	switch (direction) {
+	case "up":
+		target = (lobbies.length + (index - 1 + lobbies.length)) % lobbies.length;
+		break;
+
+	case "down":
+		target = (lobbies.length + (index + 1 + lobbies.length)) % lobbies.length;
+		break;
+	}
+
+	lobbies.eq(target).click();
+});
```

Comment:
I've been playing with this and getting a bug when switching from click to keyboard nav, and/or from jumping channels to jumping networks.
I'm not entirely sure of the sequence to repro, but it seems like on a fresh page, if I click on any channel, then go alt+shift+down (which should move me to the first network after the current channel), I jump to the top network.
Would you mind looking into it? 🙏

Reply:
OK yeah I was able to reproduce this as well… I’ll check it out.

Reply:
I’ve fixed this issue; lobby index wasn’t being calculated correctly when the active time is a channel.

However I recommend you play around with it again to see if the fixed behaviour is still desirable.

Example: if you have a channel selected, should `Alt+Shift+Up` select the lobby that channel is in (seems more intuitive UI wise) or should it select the lobby previous to the one the active channel is in (seems more intuitive hierarchy wise and is the current behaviour)

Reply:
Alright, trying now!

Reply:
Ahah, so I went testing this without reading your last paragraph to avoid bias, and got so confused why it skipped the current network 😅

I think this is a better approach:

> if you have a channel selected, should `Alt+Shift+Up` select the lobby that channel is in (seems more intuitive UI wise)

Otherwise, it feels like it skips the network. Another way to look at it is, if I'm on a channel and want to go to its network, I have to hit `Alt+Shift+Up` then `Alt+Shift+Down`.

What do you think?

Reply:
Yeah… it felt weird “skipping” to the other network. I’ll get to tweaking that behaviour sometime.